test/v1/controllers/validators/MockRetrieveOtherReliefsValidatorFactory.scala

```diff
@@ -26,16 +26,16 @@ import v1.models.request.retrieveOtherReliefs.RetrieveOtherReliefsRequestData
 
 trait MockRetrieveOtherReliefsValidatorFactory extends MockFactory {
 
-  val mockValidatorFactory: RetrieveOtherReliefsValidatorFactory = mock[RetrieveOtherReliefsValidatorFactory]
+  val mockRetrieveOtherReliefsValidatorFactory: RetrieveOtherReliefsValidatorFactory = mock[RetrieveOtherReliefsValidatorFactory]
 
-  object MockRetrieveCharitableGivingReliefValidatorFactory {
+  object MockedRetrieveCharitableGivingReliefValidatorFactory {
```

Comment:
Looks like you have named all the objects `MockedRetrieveCharitableGivingReliefValidatorFactory`. Please correct the names to be endpoint specific